.github/workflows/helm-chart-release.yml

```diff
@@ -82,3 +82,53 @@ jobs:
           publish_dir: ./to-gh-pages
           keep_files: true
           enable_jekyll: true
+  release-oci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      packages: write
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@main
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.0
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check Helm generated values are up-to-date before releasing
+        run: |
+          make check-generated-values
+
+      - name: Generate, sign and publish charts
+        env:
+          COSIGN_EXPERIMENTAL: 1
+        shell: bash
+        run: |
+          set -e
+          REGISTRY="ghcr.io/$GITHUB_REPOSITORY_OWNER"
+          charts=$(find charts/ -maxdepth 1 -mindepth 1 -type d)
+          for chart in $charts; do
+            helm package $chart
```

Comment:
Are helm charts build reproducibly? I think not, and the same chart build twice has potentially a different checksum.

If that's the case, I think we should sign the exact same helm chart artifact that is in the https repo, and not a new build. 

I wonder if chart-releaser allows to obtain the tgz apart from pushing it to gh pages, so we can upload it as a GHA artifact to reuse in this job and sign in.
If not, I wonder what would be better, a `helm pull` from the https repo, or a rebuild, as is the PR now.